app/models/manageiq/consumption/showback_charge.rb

```diff
@@ -18,7 +19,25 @@ def clean_cost
   end
 
   def stored_data_event
-    self.stored_data = self.showback_event.data
+    self.stored_data = {}
```

Comment:
Shouldn't this be self.stored_data ||= {}
?
If you call this twice you will lose all data